development/docker-compose/docker-compose.yaml

```diff
@@ -1,9 +1,10 @@
 # https://docs.docker.com/compose/compose-file/
 
 x-common-variables: &common-variables
-  JAVA_TOOL_OPTIONS: -XX:MaxRAMPercentage=60.0 -Dlogging.level.com.backbase=INFO
-  SIG_SECRET_KEY: JWTSecretKeyDontUseInProduction!
+  JAVA_TOOL_OPTIONS: -XX:MaxRAMPercentage=60.0
   USERCTX_KEY: JWTSecretKeyDontUseInProduction!
+  sso.jwt.internal.signature.key.type: VALUE
+  sso.jwt.internal.signature.key.value: JWTSecretKeyDontUseInProduction!
```

Comment:
if instead of using the SIG_SECRET_KEY env var we use the VALUE key type, it is easy for running locally services connected to the docker execution without having to add extra environment variables on the local services